bluepysnap/cli.py

```diff
@@ -22,45 +21,48 @@ def cli(verbose):
     )
 
 
-def circuit_validation_params(func):
```

Comment:
I think I did this to have the option to validate the circuit in the simulation config but opted out of implementing the feature as it seemed overly complicated. 🤔 

Anyway, happy to remove this leftover.